app/controllers/users.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const User = require('../models').user,
+  logger = require('../logger');
+
+exports.create = (req, res, next) => {
+  const params = req.body.userParams;
+  const validationOK = req.body.flagValidationOK;
+  if (validationOK === true) {
```

Comment:
Instead of comparing, you can simply do `if(validationOK)` :see_no_evil: But is this if() really necessary? Is there any situation where this flag has a `false` value? If there is, wouldn't be better to handle it in the middleware? :thinking: